<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GW2 Proxy - By Z0F</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/Xbox360-icon.ico</normaloff>assets/Xbox360-icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chk_Intercept">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Intercept</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Send">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txt_Out"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="chk_Fishing">
              <property name="text">
               <string>Easy Fishing</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chk_Zoom">
              <property name="text">
               <string>Greater Zoom</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPlainTextEdit" name="txt_Dbg"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menu_Load"/>
    <addaction name="menu_Unload"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_Load">
   <property name="icon">
    <iconset>
     <normaloff>assets/AnyConv.com__09e4ff41a8416ad38492aa52565bc81fa18ba112r1-716-344v2_uhq.ico</normaloff>assets/AnyConv.com__09e4ff41a8416ad38492aa52565bc81fa18ba112r1-716-344v2_uhq.ico</iconset>
   </property>
   <property name="text">
    <string>Load DLL</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="menu_Unload">
   <property name="text">
    <string>Unload DLL</string>
   </property>
   <property name="shortcut">
    <string>U</string>
   </property>
  </action>
  <action name="actionFishing_Toggle">
   <property name="text">
    <string>Fishing Toggle</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
